tests/imposter/data/questions.json

```diff
@@ -7,6 +7,14 @@
           "months": "3"
         }
       }
+    ],
+    "abandon": [
```

Comment:
It would be nice if we could get around this in some other way and not have to configure all the endpoints for each 'journey' in `private-api-config.yaml` 🤔